src/utils/dateUtils.ts

```diff
@@ -0,0 +1,15 @@
+export const getRelativeDays = (days: number) => {
+    if (days === 0) {
+        return "-";
+    }
+
+    if (days >= 365) {
+        return `${Math.floor(days / 365)}y`;
+    }
+
+    if (days > 30 && days < 365) {
+        return `${Math.floor(days / 30)}mo`;
+    }
+
+    return `${days}d`;
+};
```

Comment:
This really should have a test written. We can add vitest to make this work pretty quick.

```
import { getRelativeDays } from './dateUtils';

describe('getRelativeDays', () => {
  it('returns "-" for 0 days', () => {
    expect(getRelativeDays(0)).toBe('-');
  });

  it('returns "1d" for 1 day', () => {
    expect(getRelativeDays(1)).toBe('1d');
  });

  it('returns "30d" for 30 days', () => {
    expect(getRelativeDays(30)).toBe('30d');
  });

  it('returns "1mo" for 31 days', () => {
    expect(getRelativeDays(31)).toBe('1mo');
  });

  it('returns "1y" for 365 days', () => {
    expect(getRelativeDays(365)).toBe('1y');
  });

  it('returns "2y" for 730 days', () => {
    expect(getRelativeDays(730)).toBe('2y');
  });
});
```